sssoja/tv-bland
# remove unused helper file and folder

### What changes have you made?
- Removed the `decodeUrl.ts` in the helper folder 
 
### Which issue(s) does this PR fix?
Fixes #43 

### Screenshots (if there are design changes)
No design changes 

### How to test
Check that the `decodeUrl.ts` and its helper folder are gone. If we need to add a helper folder in the future, we can create a new folder in the root of the directory
